docker/Dockerfile_chatbot

```diff
@@ -0,0 +1,145 @@
+    wget numactl git openssh-server ca-certificates cmake && \
+    apt-get clean autoclean && \
+    apt-get autoremove -y && \
+    rm -rf /var/lib/apt/lists/*
+
+# Download ITREX code
+#RUN git clone --single-branch --branch=${ITREX_VER} ${REPO} intel-extension-for-transformers
+RUN mkdir -p /intel-extension-for-transformers
+WORKDIR /intel-extension-for-transformers
+COPY . /intel-extension-for-transformers
+
+# Install miniconda
+ENV CONDA_DIR /opt/conda
+RUN wget --quiet https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh -O ~/miniconda.sh && \
+    /bin/bash ~/miniconda.sh -b -p /opt/conda
+
+# Put conda in path so we can use conda activate
+SHELL ["/bin/bash", "--login", "-c"]
+ENV PATH=$CONDA_DIR/bin:$PATH
+RUN conda init bash && \
+    unset -f conda && \
+    export PATH=$CONDA_DIR/bin/:${PATH} && \
+    conda config --add channels intel && \
+    conda create -yn neuralchat python=${PYTHON_VERSION} && \
+    echo "conda activate neuralchat" >> ~/.bashrc && \
+    source ~/.bashrc
+
+RUN source activate && conda activate neuralchat && pip install oneccl_bind_pt -f https://developer.intel.com/ipex-whl-stable-cpu && \
+    cd /intel-extension-for-transformers && pip install -r requirements.txt && pip install -v . && \
+    cd ./intel_extension_for_transformers/neural_chat/examples/instruction_tuning && pip install -r requirements.txt && \
+    cd /intel-extension-for-transformers/intel_extension_for_transformers/neural_chat && pip install -r requirements_cpu.txt && \
+    cd /intel-extension-for-transformers/examples/huggingface/pytorch/code-generation/quantization && pip install -r requirements.txt --no-cache-dir && \
+    conda install astunparse ninja pyyaml mkl mkl-include setuptools cmake cffi typing_extensions future six requests dataclasses -y && \
+    conda install jemalloc gperftools -c conda-forge -y && \
+    pip install datasets accelerate SentencePiece evaluate nltk rouge_score protobuf==3.20.2 tokenizers einops peft && \
+    pip install torch==2.1.0+cpu -f https://download.pytorch.org/whl/cpu/torch_stable.html && \
+    pip install /intel-extension-for-transformers/intel_extension_for_pytorch-2.1.0-cp310-cp310-manylinux2014_x86_64.whl && \
+    pip install torchaudio==2.1.0+cpu torchvision==0.16.0+cpu -f https://download.pytorch.org/whl/cpu/torch_stable.html
+
+# Enable passwordless ssh for mpirun
+RUN mkdir /var/run/sshd
+RUN passwd -d root
+RUN sed -i'' -e's/^#PermitRootLogin prohibit-password$/PermitRootLogin yes/' /etc/ssh/sshd_config \
+        && sed -i'' -e's/^#PasswordAuthentication yes$/PasswordAuthentication yes/' /etc/ssh/sshd_config \
+        && sed -i'' -e's/^#PermitEmptyPasswords no$/PermitEmptyPasswords yes/' /etc/ssh/sshd_config \
+        && sed -i'' -e's/^UsePAM yes/UsePAM no/' /etc/ssh/sshd_config \
+        && echo "StrictHostKeyChecking no" >> /etc/ssh/ssh_config
+EXPOSE 22
+
+WORKDIR /intel-extension-for-transformers/
+
+CMD ["/usr/sbin/sshd", "-D"]
+
+# HABANA environment
```

Comment:
should we create 2 dockers for habana and cpu?

Reply:
It's okay we keep havana stage. Only if we need habana image, we will use --target hpu to create it.